challenges.yml

```diff
@@ -65,3 +67,19 @@ stimulus_controllers_convention:
         c: |
           It will connect to the `api_manager.js` file
       correct: b
+stimulus_targets:
+  question: |
+    What's the main purpose of Stimulus targets?
```

Comment:
This one seems too easy / high level...

Instead, we could show them us registering a target in the controller, then asking what the HTML would look like to attach that target. Or we could show them a functional target that's in a loop and ask them what we would need to change to be able to get an array of all the matched targets in the controller instead of just one.